### PR TITLE
Add Arktype Validator

### PR DESCRIPTION
**Pull request by Makisuo**

### CODE REVIEW

File: examples/react/arktype/index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" type="image/svg+xml" href="/emblem-light.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+
+    <title>TanStack Form React Valibot Example App</title>
```
Comment (Pascalmh) [inline examples/react/arktype/index.html:9]:
```suggestion
    <title>TanStack Form React Arktype Example App</title>
```

File: examples/react/arktype/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      // Do something with form data
+      console.log(value)
+    },
+    // Add a validator to support Arktype usage in Form and Field
+    validatorAdapter: arktypeValidator(),
+  })
+
+  return (
+    <div>
+      <h1>Valibot Form Example</h1>
```
Comment (Pascalmh) [inline examples/react/arktype/src/index.tsx:34]:
```suggestion
      <h1>Arktype Form Example</h1>
```

File: examples/react/arktype/package.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    "preview": "vite preview",
+    "test:types": "tsc"
+  },
+  "dependencies": {
+    "@tanstack/arktype-form-adapter": "^0.28.0",
+    "@tanstack/react-form": "^0.29.1",
+    "arktype": "2.0.0-beta.5",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
+    "valibot": "^0.37.0"
```
Comment (Pascalmh) [inline examples/react/arktype/package.json:17]: The `"valibot"` dependency should not be needed here

**Review by Pascalmh — CHANGES_REQUESTED** [review]

**Comment by Balastrong** [comment]
Thank you everyone for the effort here!
The bad news is that I'm closing this PR, the good news is that after #1020 lands, arktype will be natively supported as it implements Standard Schema 🙌

**Pull request closed without merging**